appeals/web/src/server/appeals/appeal-details/interested-party-comments/view-and-review/view-and-review.validators.js

```diff
@@ -2,5 +2,8 @@ import { createValidator } from '@pins/express';
 import { body } from 'express-validator';
 
 export const validateReviewComment = createValidator(
-	body('status').notEmpty().isIn(['valid', 'invalid']).withMessage('Something went wrong')
+	body('status')
+		.notEmpty()
+		.isIn(['valid', 'invalid', 'valid_requires_redaction'])
+		.withMessage('Something went wrong')
```

Comment:
This error message may be should be clarified to make it clearer what went wrong (ie one option must be selected)

Reply:
I agree the message could be clearer but this isn't really in the scope of this ticket. I just needed to add this option to the `isIn` to progress the journey and the code got reformatted as a result